src/components/ellipsis/ellipsis.tsx

```diff
@@ -220,7 +222,15 @@ export const Ellipsis: FC<EllipsisProps> = p => {
 
   return withNativeProps(
     props,
-    <div ref={rootRef} className={classPrefix}>
+    <div
+      ref={rootRef}
+      className={classPrefix}
+      onClick={e => {
+        if (e.target === e.currentTarget) {
+          props?.onContentClick(e)
```

Comment:
这里 props 应该不用 `?.` 的

```suggestion
          props.onContentClick(e)
```